src/tests/end-to-end/common/page.ts

```diff
@@ -9,18 +9,22 @@ import { DEFAULT_NEW_PAGE_WAIT_TIMEOUT_MS, DEFAULT_PAGE_ELEMENT_WAIT_TIMEOUT_MS
 export class Page {
     constructor(private readonly underlyingPage: Puppeteer.Page) {
         underlyingPage.on('error', error => {
-            forceTestFailure(`error occurred - ${error.message}`);
+            forceTestFailure(`Puppeteer.Page '${underlyingPage.url()}' emitted 'error' with stack: ${error.stack}`);
```

Comment:
Do we miss anything by not including `error.message` here?

Reply:
Just  more context to `error.message`. It was basically just saying `Page crashed` in case of a page crash. Maybe adding it in conjecture with the current `error.stack` is helpful.

Reply:
error.stack already contains error.name and error.message as its first line